src/mainviews/edit-job-definition.tsx

```diff
@@ -89,6 +93,11 @@ function EditJobDefinitionBody(props: EditJobDefinitionProps): JSX.Element {
           {displayError}
         </Alert>
       )}
+      <InputFileSnapshot inputFileSnapshot={props.model.inputFileSnapshot} />
+      <LabeledValue
+        value={timestampLocalize(props.model.updateTime)}
```

Comment:
I'm still not clear on this: Does "Updated at" / `updateTime` mean:

1. The time at which the **job definition** was last updated?
2. The time at which the **input file** was last updated on disk?

At the very least, we should have meaningful and unambiguous variable names and comments to ensure that future maintainers understand this.

Reply:
It's #1. So it will have a new value any time JobDefinition gets updated on the backend (it's mapped from backend JobDefinition field `update_time`)

Reply:
Moved labeled value to be below the page title / above the Input file snapshot field, changed label text to `Job Definition updated at` to make labeled value's meaning more clear. @jweill-aws, could you please take a look and see if it it seems clearer to you now? 

![Screen Shot 2022-11-17 at 10 50 33 AM](https://user-images.githubusercontent.com/26686070/202533571-afa23d15-1bc0-46f3-af27-b5e22f690d3e.png)